src/plugin/fieldViews/AttributeFieldView.ts

```diff
@@ -53,9 +53,9 @@ export abstract class AttributeFieldView<Value extends unknown>
     return true;
   }
 
-  public update(value: Value) {
+  public update = (value: Value) => {
```

Comment:
Why have these been changed to arrow functions?

Reply:
Having this function lexically scoped means we can pass it around without worrying about it being bound to the wrong thing – e.g. L23 of src/renderers/react/useCustomFieldViewState.ts.  

Having said that, this is a bit obscure. Torn as to whether to wrap the invocation on L23 in another arrow function and restore the old version. Not much in it.

Reply:
Of course, I missed that. I think it would be preferable to keep our functions consistent in the class and wrap it as you described, it would mean you only have use that syntax in one place rather than in each class. However, I'm not too fussy.

Reply:
Agreed, I'll do that – thanks for the feedback.